Utils/functions.py

```diff
@@ -0,0 +1,101 @@
+import re
```

Comment:
Sort import in next order:
1. blocks `import` and `from ... import`;
2. alphabetical order within each block;

https://www.python.org/dev/peps/pep-0008/#imports